### PR TITLE
Redirect CAS users to cas login upon accepting invitation

### PR DESCRIPTION
**Pull request by aitorlb**
Related issue: #46

### CODE REVIEW

File: spec/system/participatory_space_private_user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+    context "when inviting CAS user" do
+      let(:fill_cas_user_checkbox) do
+        check :participatory_space_private_user_cas_user
+      end
+
+      include_examples "delivered email has correct invitation link"
+
+      include_examples "following invitation link" do
+        let(:expected_path) { "/users/cas/sign_in" }
```
Comment (microstudi) [inline spec/system/participatory_space_private_user_spec.rb:89]: This expected path, when using a real CAS server is converted to another thing: `something/login?return...`

**Review by microstudi — COMMENTED** [review]
Thanks for this!
I've tried to run the test and it seems that a working CAS server is needed right?

I had to define `export CAS_BASE_URL="somenergia_cas_server_url"` before running the specs.

I think we should add some documentation for this requirement in the README.md. Or maybe try to stub the CAS server URL. but I've been unable to achieve it in my tests.

In the comments there's another issue related to this.

File: app/decorators/decidim/devise/invitations_controller_decorator.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+Decidim::Devise::InvitationsController.class_eval do
+  # GET /resource/invitation/accept?invitation_token=abcdef
+  def edit
+    set_minimum_password_length
+    resource.invitation_token = params[:invitation_token]
+
+    if cas_user?
+      redirect_to cas_user_path
```
[the remark this answers is not in the document]
Comment (aitorlb) [reply app/decorators/decidim/devise/invitations_controller_decorator.rb]: Yes, it does. Thanks.

File: spec/system/participatory_space_private_user_spec.rb (hunk 1 above)
Comment (aitorlb) [reply spec/system/participatory_space_private_user_spec.rb]: Updated the test. It should pass both in local and when using a real CAS server.

**Review by microstudi — APPROVED** [review]
Thanks!!

**Pull request merged**